.github/CONTRIBUTING.md

```diff
@@ -54,13 +54,14 @@ gh repo clone wagmi-dev/abitype
 
 ## Installing Node.js and pnpm
 
-ABIType uses [pnpm](https://pnpm.io) as its package manager. You need to install **Node.js v16 or higher** and **pnpm v7 or higher**.
+ABIType uses [pnpm](https://pnpm.io) as its package manager. You need to install **Node.js v16 or higher**, **pnpm exactly v8.3.1** and **TypeScript exactly v5.0.4**.
```

Comment:
Sorry my wording in the previous comment was incorrect. I was more thinking in adding a "Note" for anyone that might want to install packages. Something like this.

> **Note:** In case you have to install new packages or upgrade packages make sure to use **pnpm@8.3.1** and **typescript@5.0.4**

You can revert the message to the older one and add this note

Reply:
Okay, no problem!